ext/readline/readline.c

```diff
@@ -262,6 +266,10 @@ PHP_FUNCTION(readline_info)
 		add_assoc_long(return_value,"pending_input",rl_pending_input);
 		add_assoc_string(return_value,"prompt",SAFE_STRING(rl_prompt));
 		add_assoc_string(return_value,"terminal_name",(char *)SAFE_STRING(rl_terminal_name));
+		completion_append_str[0] = (char)rl_completion_append_character;
+		completion_append_str[1] = '\0';
+		add_assoc_string(return_value,"completion_append_character",SAFE_STRING(completion_append_str));
+		add_assoc_long(return_value,"completion_suppress_append",rl_completion_suppress_append);
```

Comment:
Wouldn't it make more sense to return this as a boolean (and similarly set it as a boolean)?

Reply:
That would make more sense! I'll change it.